zomes/holo_acorn/code/src/profile/mod.rs

```diff
@@ -152,6 +159,24 @@ pub fn update_whoami(profile: Profile, address: Address) -> ZomeApiResult<GetRes
         address: address,
     })
 }
+pub fn update_status(status: Status) -> ZomeApiResult<GetResponse<Profile>> {
+    if let Some(GetResponse { entry, address }) = whoami()? {
+        update_whoami(
+            Profile {
+                first_name: entry.first_name,
+                last_name: entry.last_name,
+                handle: entry.handle,
+                status: status,
+
+                avatar_url: entry.avatar_url,
+                address: entry.address,
+            },
+            address,
+        )
+    } else {
+        Err(ZomeApiError::Internal("error".into()))
+    }
+}
```

Comment:
is there a reason we wouldn't just use `update_whoami` instead of adding a new one?

Reply:
you know what, I'm fine with this 👍